packages/eas-cli/src/credentials/ios/appstore/experimental.ts

```diff
@@ -0,0 +1,26 @@
+import { BundleId, Profile } from '@expo/apple-utils';
+import { boolish } from 'getenv';
+
+export const USE_APPLE_UTILS = boolish('USE_APPLE_UTILS', false);
+
+export async function getProfilesForBundleIdAsync(bundleIdentifier: string): Promise<Profile[]> {
+  const [bundleId] = await BundleId.getAsync({
+    query: {
+      filter: {
+        identifier: bundleIdentifier,
+      },
+    },
+  });
```

Comment:
i've seen this query multiple times, it'd be nice if this was a helper so we don't have to destructure all the time (ie) `getByExplicitIdentifierAsync(explicitBundleIdentifier: string): Promise<BundleId>`